vectorization/simple_vectorization.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+
+
+def simpleVectorization(spectrum):
```

Comment:
definitely try the naive vectorization where you bin the spectra. 
So go from mz 20 to mz 1000 or so and bin in 0.05 windows or similar. So just add all the intensities within that window to a number.
Not sure what input milvus and other databases support but the intensities could be reduced in precision.